ci_scripts/build_documentation.sh

```diff
@@ -2,31 +2,37 @@
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
+function info {
+  echo "[$(basename ${0})] [INFO] ${1}"
+}
+
+function die {
+  echo "[$(basename ${0})] [ERROR] ${1}"
```

Comment:
I'm sure it's fine, and now I'm not sure what would be more convenient for running and viewing script output.

If I were going to do it, maybe something like `echo "..." | tee /dev/stderr`